RockWeb/Plugins/org_lakepointe/CheckIn/Labels/Child_Label_and_Slips_v2.lava

```diff
@@ -101,12 +102,16 @@ CT~~CD,~CC^~CT~
 
 ^FO560,340^FB290,1,0,C^A0N,60,60^FD{{ Person.SecurityCode }}^FS
 
-^FO7,363^FB67,1,0,L^A0N,43,43^FH\^FD{% if personAllergy != '' %}A{% endif %}^FS
-^FO73,363^FB67,1,0,L^A0N,43,43^FH\^FD{% if personMedicalAlert  != '' %}M{% endif %}^FS
-^FO140,363^FB67,1,0,L^A0N,43,43^FH\^FD{% if personLegalNotes != '' %}C{% endif %}^FS
-^FO207,363^FB67,1,0,L^A0N,43,43^FH\^FD{% if personBaptized  != ''  %}B{% endif %}^FS
-^FO273,363^FB67,1,0,L^A0N,43,43^FH\^FD{% if personFoster == 'true' %}P{% endif %}^FS
-^FO340,363^FB67,1,0,L^A0N,43,43^FH\^FD{% if personSelfRelease == 'Yes' and isSelfReleaseEligible == true %}S{% endif %}^FS
+^FO10,355^FB67,1,0,L^A0N,43,43^FH\^FD{% if personAllergy != '' %}A{% endif %}^FS
+^FO70,355^FB67,1,0,L^A0N,43,43^FH\^FD{% if personMedicalAlert  != '' or personMedicalAlert2 == 'Yes' %}M{% endif %}^FS
+^FO140,355^FB67,1,0,L^A0N,43,43^FH\^FD{% if personLegalNotes != '' %}C{% endif %}^FS
+^FO200,355^FB67,1,0,L^A0N,43,43^FH\^FD{% if personBaptized  != ''  %}B{% endif %}^FS
+^FO270,355^FB67,1,0,L^A0N,43,43^FH\^FD{% if personFoster == 'true' %}P{% endif %}^FS
+^FO340,355^FB67,1,0,L^A0N,43,43^FH\^FD{% if personSelfRelease == 'Yes' and isSelfReleaseEligible == true %}S{% endif %}^FS
+
+{% if Person.DaysToBirthday < 7 %}
+^FO470,330^GFA,201,792,12,:Z64:eJzVzjEKwzAMBVAbDR51BF2jQ2guFmiOlqPkCN4qqJFqxzJJKaakS6kE5g2S9d3F7TW4CLP5mr00ew6rWYDR7DUwHTx+suzzrrj9P/kV290JIi7/a9JcUn0r1vnNG7UubKO2nMoTqvmkHxEi3Ks1YUL7X4WEmg93X0y5R8tDgmJ5OJT+Lk/P3Qwd/6qe5tKwVw==:52D8
```

Comment:
New rendered label:
![image](https://github.com/user-attachments/assets/6afe639f-aeb8-46f2-9a7d-61728ed43751)

Added some additional padding around the letters just to help out the label printers